Controller/Component/WizardComponent.php

```diff
@@ -157,6 +157,13 @@ class WizardComponent extends Component {
 	public $nestedViews = false;
 
 /**
+ * Holds the root of the session key for data storage.
+ *
+ * @var string
+ */
+	public $sessionRootKey = 'Wizard';
```

Comment:
Okay. I was just wondering what sort of ways it could automatically be set once and work with both the component and helper.